clients/shared/services/authService/index.ts

```diff
@@ -0,0 +1,10 @@
+import { post } from '../api'
+import { AxiosResponse } from 'axios'
+
+const authService = {
+  registerOrganization: async(data: object): Promise<AxiosResponse<any>> => {
+    return await post('/auth/register-organization', data)
```

Comment:
no need to await

Reply:
updated in a3d06124eeaeb1aff9ed3fd02c4eadf7f7467947

Reply:
its giving linting error if we don't use await